thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/detection/AnomalyDetectionInputContext.java

```diff
@@ -13,19 +13,19 @@
 
 
 public class AnomalyDetectionInputContext {
-  Map<DimensionMap, MetricTimeSeries> dimensionKeyMetricTimeSeriesMap = Collections.emptyMap();
+  Map<DimensionMap, MetricTimeSeries> dimensionMapMetricTimeSeriesMap = Collections.emptyMap();
```

Comment:
I think it is necessary. Using dimensionKeyMetricTimeSeriesMap is misleading. We should remain the consistency with the key-value type with the name.